ckan/lib/helpers.py

```diff
@@ -1520,11 +1511,39 @@ def gravatar(email_hash, size=100, default=None):
         default = quote(default, safe='')
 
     return literal('''<img src="//gravatar.com/avatar/%s?s=%d&amp;d=%s"
-        class="gravatar" width="%s" height="%s" alt="Gravatar" />'''
+        class="user-image" width="%s" height="%s" alt="Gravatar" />'''
                    % (email_hash, size, default, size, size)
                    )
 
 
+@core_helper
+def user_image(user_id, size=100):
+    try:
+        user_dict = logic.get_action('user_show')(
+            {'ignore_auth': True},
+            {'id': user_id}
+        )
+    except logic.NotFound:
+        return ''
+
+    gravatar_default = config.get('ckan.gravatar_default', 'identicon')
+
+    if user_dict['image_display_url']:
+        return literal('''<img src="{url}"
+                       class="user-image"
+                       width="{size}" height="{size}" alt="{alt}" />'''.format(
+            url=user_dict['image_display_url'],
+            size=size,
+            alt=user_dict['name']
+        ))
```

Comment:
Not a change-request. Just want to mention that i've added `dominate` library during migration to py3 and it allows following constructions:

```
img = literal(dom_tags.img(
    src="...", cls="...", width=100, height=100
))
# or, when you have more complex nested structer, `with` blocks can be used
with dom_tags.img() as img:
    dom_tags.attr(src="...")
    dom_tags.attr(cls="...")
    dom_tags.attr(width=100, height=100)
img = literal(img)
```

Reply:
Good to know thanks!